exercises/array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,13 @@
+/* exported includes */
+function includes(array, value) {
+  if (array.length === 0) {
+    return false;
+  }
```

Comment:
You can remove this conditional. If the array is empty, the loop would never start, which leaves you in the same situation once the function ends